Gemfile.lock

```diff
@@ -414,8 +414,5 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   underscore-rails
 
-RUBY VERSION
```

Comment:
Oops, didn't mean to commit that. My Gemfile.lock keeps changing this for some reason, so I usually have to `git checkout Gemfile.lock` to avoid committing it. I should probably figure out why that's happening. 

Good catch 👍